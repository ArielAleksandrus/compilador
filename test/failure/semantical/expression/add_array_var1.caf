programa{
	int a[30], b = 3, c = a + 3;
}