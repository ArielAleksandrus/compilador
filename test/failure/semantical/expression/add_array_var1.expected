ERROR => Line 2, col 24: Both lval and rval should be of type int or car. Found: INT ARRAY and INT
